fitting/interpolation.py

```diff
@@ -18,7 +18,7 @@
 
 args = parser.parse_args()
 ### use same settings as in diboson search
-all_graphs = {"graviton" : "mean:spline,sigma:spline,alpha:pol1,sign:pol1,alpha2:pol2,sign2:pol2"}
+all_graphs = {"diboson" : "mean:spline,sigma:spline,alpha:pol1,sign:pol1,alpha2:pol2,sign2:pol2"}
```

Comment:
This is gonna break our scripts, can we keep it unchanged?

Reply:
Ok sure, I just got confused by the naming but I can change it back